EventsExpress/ClientApp/src/services/EventsExpressService.js

```diff
@@ -10,14 +10,14 @@ export default class EventsExpressService {
         const res = await this.getResource('chat/GetAllChats');
         return res;
     }
-    
+
     getChat = async (chatId) => {
         const res = await this.getResource(`chat/GetChat?chatId=${chatId}`);
         return res;
     }
 
     getEvents = async (eventIds, page) => {
-        const res = await this.setResource('event/getEvents?page='+page, eventIds);
+        const res = await this.setResource('event/getEvents?page=' + page, eventIds);
```

Comment:
Please use string interpolation rather than concatenation and in all of the methods below.